### ISSUE TITLE
fix accessibility issues

### ISSUE THREAD

**Issue by Vincenius** [Vincenius/workout-lol#53]
The lighthouse score shows some accessibility errors. Those should be fixed!

Maybe also use another tool to test for more errors (eg. https://wave.webaim.org/)

**Comment by ngthuongdoan** [comment]
@Vincenius can you open the assignees option on issue, to prevent duplicate of works

**Comment by Vincenius** [comment]
Hey :) I couldn't find the option to open assignees on issues. But I invited you as collaborator - I guess that should work as well 🤔
